scripts/qa_crowbarsetup.sh

```diff
@@ -2365,6 +2365,10 @@ function oncontroller_testsetup()
         fi
     fi
 
+    #test for Glance scrubber service, added after bnc#930739
+    su - glance -s /bin/sh -c "/usr/bin/glance-scrubber --config-dir /etc/glance" || \
+    complain 113 "Glance scrubber doesn't work properly"
```

Comment:
:-1: on how it is atm with glance-scrubber always returning 0, this change is useless.
Also we don't actually test the cron entry, but a one-time-copy of it, so we would miss any future breakage from changes in it.